GDevelop.js/__tests__/GDJSSceneVariableExtensionCodeGenerationIntegrationTests.js

```diff
@@ -0,0 +1,489 @@
+const initializeGDevelopJs = require('../../Binaries/embuild/GDevelop.js/libGD.js');
```

Comment:
(i think you can remove "Extension" from the filename, it's such a core feature that it can as well be just named "Variable" in these test files)